src/htdocs/storm/storm12.php

```diff
@@ -0,0 +1,114 @@
+    –Dst = 66 nT at 23 January at 00:45. NOAA reported a Kp= 5 magnetic index,
+    a generic measure of mid-latitude activity for a single 3-hr period on 22
+    January, corresponding to a “minor” magnetic storm. It is noteworthy that
+    at high-latitudes magnetic disturbance can be much greater in amplitude and
+    intensity. The direction of the magnetic field at Barrow, Alaska, the most
+    northerly USGS observatory, changed by about 6.0 degrees over a period of
+    15 minutes. Such variations, large enough to be seen on an ordinary
+    compass, would have been a nuisance for directional-drilling operations on
+    the North Slope. This storm also produced beautiful displays of aurora at
+    high latitudes. As we leave the quiet period of the last solar minimum and
+    head into the next solar maximum, predicted for 2013, and over the next 3
+    years or so we can expect more storms, some of which will probably be much
+    larger than the 23 January 2012 storm.
+  </p>
+
+  <p>
+    On 23 January, another solar flare and CME was observed. This might cause
+    another magnetic storm at Earth on late 24 January.
+  </p>
+
+  <p>
+    For perspective: The largest storm of the 20th century occurred on March,
+    1989 -Dst = 574 nT. This storm induced electric currents in the Earth’s
+    crust that found their way through ground connections into the high-voltage
+    Canadian Hydro-Québec power grid, causing transformer failure and resulting
+    in the loss of electric power to more than 6 million people. The same
+    storm also damaged and disrupted the operation of satellites and it
+    severely disrupted GPS systems and over-the-horizon radio communication
+    systems used by the US military. The storm of March 1989, large though it
+    was, pales in comparison to that of September 1859, the largest storm in
+    recorded history occurred, -Dst = 1600 nT (approximately). If such a storm
+    were to occur today, the economic impact to the United States because of
+    disrupted technological systems could exceed $1 trillion
+    (Baker et al., 2008).
+  </p>
+
+  <p>
+    In addition to supporting internal USGS operational needs, real-time USGS
+    data are used by NOAA’s Spaee Weather Prediciton Center, the Air Force
+    Weather Agency, the Air Force Research Laboratory, NASA’s Goddard Space
+    Flight Center, Japan’s National Institute of Information and Communications
+    Technology, and several private companies for space weather diagnostics and
+    directional drilling.
+  </p>
+
+  <ul>
+    <li>Additional information:</li>
+    <li>
+      USGS Geomagnetic Dst: <a href="geomag.usgs.gov/dst/">
+      geomag.usgs.gov/dst/</a>
+    </li>
+    <li>
+      Intermagnet data: <a href="www.intermagnet.org">www.intermagnet.org</a>
+    </li>
+    <li>
+      Solar images: National Solar Observatory
+      <a href="http://www.nso.edu/current_images.html">
```

Comment:
For these, check that `https://` does not work. If it does not work, it is fine to use `http://`, otherwise, switch to `https://`